### DISCUSSION TITLE
Offline editor version?

### DISCUSSION THREAD

**Discussion by the2000isawesome** [ScratchAddons/ScratchAddons#1906] [Help]
I use the offline editor and I'm wondering if anyone has compiled a version of the offline editor with SA features. Some things such as onion skinning and the clone count appeal to me, but I can't use the online editor because I have an old computer and it lags too much.

**Comment by cobaltt7** [comment]
Yes, see here: https://desktop.turbowarp.org/

**Reply by cobaltt7** [reply]
In response to your lag problem, TW converts projects to JS so it'll run faster

**Comment by retronbv** [comment]
You could also just use the turbowarp website

**Reply by cobaltt7** [reply]
...that's not offline...

**Reply by retronbv** [reply]
but it will remove their lag problem

**Reply by cobaltt7** [reply]
they never said they had lag on the website

**Reply by retronbv** [reply]
Yes they did?

**Reply by retronbv** [reply]
> but I can't use the online editor because I have an old computer and it lags too much.

**Reply by cobaltt7** [reply]
oh i misread i thought they said "I can't use the offline editor"

**Comment by WorldLanguages** [comment]
IMO the offline editor will lag the same or more than the online editor, if that's what you're looking for.

**Reply by the2000isawesome** [reply]
I use the offline editor and it runs much faster for me, so I don't think that's true.

**Reply by WorldLanguages** [reply]
Huh, I wonder why. Scratch offline is just a browser that can only run Scratch.
